src/main/java/bio/terra/service/resourcemanagement/google/GoogleBucketService.java

```diff
@@ -282,11 +287,18 @@ public void updateBucketMetadata(String bucketName, String flightId) {
    * @param bucketResource description of the bucket resource to be created
    * @return a reference to the bucket as a GCS Bucket object
    */
-  private Bucket newCloudBucket(GoogleBucketResource bucketResource) {
+  private Bucket newCloudBucket(GoogleBucketResource bucketResource, Integer daysToLive) {
     boolean doVersioning =
         Arrays.stream(env.getActiveProfiles()).noneMatch(env -> env.contains("test"));
     String bucketName = bucketResource.getName();
     GoogleRegion region = bucketResource.getRegion();
+    List<BucketInfo.LifecycleRule> lifecycleRules = new ArrayList<>();
+    if (daysToLive != null) {
```

Comment:
wait...does this actually delete the bucket?  Or just the contents in the bucket?

Reply:
nope just objects.